scripts/build.ps1

```diff
@@ -55,13 +55,16 @@ Param(
 
     # Using this option will skip all downloads, if you only need to build launcher
     [switch]$noVendor,
+    
+    # Using this option will specify the emulator to use [all, conemu-maximus5, or windows-terminal]
+    [string]$emulator = 'all',
 
     # Build launcher if you have MSBuild tools installed
     [switch]$Compile
 )
 
 # Get the scripts and cmder root dirs we are building in.
-$cmder_root = Resolve-Path "$PSScriptRoot\.."
+$cmder_root = [string](Resolve-Path "$PSScriptRoot\..")
```

Comment:
I approve but just curious, do you remember what issue did you run into that required a cast to string?

Reply:
No - Removed